src/Unleash/DefaultUnleash.cs

```diff
@@ -30,22 +30,34 @@ public class DefaultUnleash : IUnleash
         private readonly UnleashServices services;
 
         ///// <summary>
-        ///// Initializes a new instance of Unleash client. 
+        ///// Initializes a new instance of Unleash client with a set of default strategies. 
         ///// </summary>
         ///// <param name="config">Unleash settings</param>
-        ///// <param name="strategies">Available strategies. When none defined, all default strategies will be added.</param>
+        ///// <param name="strategies">Additional custom strategies.</param>
         public DefaultUnleash(UnleashSettings settings, params IStrategy[] strategies)
+            : this(settings, overrideDefaultStrategies: false, strategies)
+        { }
+
+        ///// <summary>
+        ///// Initializes a new instance of Unleash client.
+        ///// </summary>
+        ///// <param name="config">Unleash settings</param>
+        ///// <param name="overrideDefaultStrategies">When true, it overrides the default strategies.</param>
+        ///// <param name="strategies">Custom strategies.</param>
+        public DefaultUnleash(UnleashSettings settings, bool overrideDefaultStrategies, params IStrategy[] strategies)
```

Comment:
I'm not sure if I understand the use case for the `overrideDefaultStrategies`? 

Other client SDKs has implemented this in the following way:
The client has a list of strategies it understand. If the user sends in a custom implementation with the same name as an default strategy the custom strategy will be used.

Reply:
I mostly added it so that people experiencing breaking changes, could fix it by changing their code to:
```C#
var unleash = new DefaultUnleash(settings, true, myStrategy, myOtherStrategy);
```
thus having it behave as it did before. Basically a means to make the breaking changes more acceptable - although, I'm not sure how big of a deal these changes really are.

I do like your suggestion.